src/status_im/ui/screens/wallet/db.cljs

```diff
@@ -24,9 +24,12 @@
   (let [amount' (string/replace amount #"," ".")
         amount-splited (string/split amount' #"[.]")]
     (cond
-      (or (nil? amount) (= amount "") (= amount "0") (re-matches #"0[,.]0*$" amount))
+      (or (nil? amount) (= amount "") (re-matches #"0[,.]0*$" amount))
       nil
 
+      (= amount "0")
```

Comment:
zero should be valid, later user can send data with 0 amount

Reply:
We don't allow to provide data for now so I'd rather let this and revisit when we add it.